assignment-sweet-compose/answer/docker-compose.yml

```diff
@@ -56,11 +55,16 @@ services:
       - traefik.frontend.rule=Host:vote.localhost
 
   result:
-    build:
-      context: ../result
-    command: nodemon index.js
+    build: 
+      context: result
+    command: nodemon --inspect=0.0.0.0:9229 server.js
+    environment:
+      - NODE_ENV=development
+    ports:
+      - '5001:80'
+      - '9229:9229'
     volumes:
-      - ../result:/app
+      - ./result:/app
```

Comment:
Here it is a little bit tricky. If we consider we copy and paste the file from `./answer/docker-compose.yml` folder to the `./docker-compose.yml` - everything is OK. But if we need to launch compose using `docker-compose -f  ./answer/docker-compose.yml up` it will have broken context path

Reply:
The answer file says at the top to run it from the answer directory.